extension/oauth2clientauthextension/extension.go

```diff
@@ -76,6 +76,7 @@ func newClientCredentialsExtension(cfg *Config, logger *zap.Logger) (*ClientCred
 			ClientSecret: cfg.ClientSecret,
 			TokenURL:     cfg.TokenURL,
 			Scopes:       cfg.Scopes,
+			EndpointParams: cfg.EndpointParams,
```

Comment:
Thanks @tomsanbear. Can you please modify this test case as well https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/oauth2clientauthextension/extension_test.go#L48 in addition to the configuration test you modified. This test is a proper e2e use case.

Everything else LGTM

Reply:
Hey @pavankrish123 , thanks for the quick review, i've pushed the test up to the branch. Let me know if there is anything else you suggest to add :+1:

Reply:
Thank you. I have added a minor suggestion on the test. Please add the test and everything else LGTM